CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,4 @@
+- Fix transport in autoprovision device depending on binding [#235]
+- Allow get list of commands without sending measures (empty payload) [#234]
+- Add: allow NGSIv2 for updating active attributes at CB, throught configuration based on iotagent-node-lib(#233)
+
```

Comment:
Lines 1 and 2 should be removed (there were already included in release 1.6.0, see https://github.com/telefonicaid/iotagent-ul/releases/tag/1.6.0)

Reply:
Permission file changed to 644